cmake/CMakeLists.txt

```diff
@@ -1417,10 +1417,14 @@ if (onnxruntime_USE_TVM)
     set(USE_CUDA ON CACHE BOOL "Only defined for TVM" FORCE)
   endif()
 
+  add_definitions(-DTVM_LOG_CUSTOMIZE=1)
+  add_library(custom_logger STATIC ${ONNXRUNTIME_ROOT}/core/providers/tvm/custom_logging.cc)
```

Comment:
can it be called tvm_custom_logger instead?

Reply:
done